tests/Doc/StringSnippets.cs

```diff
@@ -27,7 +27,7 @@ public void run()
 
         // STEP_START set_get
         var res1 = db.StringSet("bike:1", "Deimos");
-        Console.WriteLine(res1); // true
+        Console.WriteLine(res1.ToString()); // true
```

Comment:
Isn't `ToString()` implied here? These code snippets end up on the reddis.io web documentation, they should be as easy to read as possible.

Reply:
Hello Gabriel,
How are you?
Thanks for the points you mentioned.
I checked with branch 'master' and found that '.ToString()' wasn't implemented, but I don't think it would cause any issues since it just logs.
Your opinion is still a priority, and if you agree, we can make some changes.